ios/FluentUI/Pill Button Bar/PillButtonTokens.swift

```diff
@@ -5,101 +5,121 @@
 
 import UIKit
 
-class MSFPillButtonTokens: MSFTokensBase {
-    public var backgroundColor: UIColor!
-    public var titleColor: UIColor!
-
-    public var hoverBackgroundColor: UIColor!
-
-    public var selectedBackgroundColor: UIColor!
-    public var selectedTitleColor: UIColor!
-
-    public var disabledBackgroundColor: UIColor!
-    public var disabledTitleColor: UIColor!
-
-    public var selectedDisabledBackgroundColor: UIColor!
-    public var selectedDisabledTitleColor: UIColor!
-
-    public var highlightedBackgroundColor: UIColor!
-    public var highlightedTitleColor: UIColor!
-
-    public var selectedHighlightedBackgroundColor: UIColor!
-    public var selectedHighlightedTitleColor: UIColor!
-
-    public var enabledUnreadDotColor: UIColor!
-    public var disabledUnreadDotColor: UIColor!
-
-    public var bottomInset: CGFloat!
-    public var font: UIFont!
-    public var horizontalInset: CGFloat!
-    public var topInset: CGFloat!
-    public var unreadDotOffsetX: CGFloat!
-    public var unreadDotOffsetY: CGFloat!
-    public var unreadDotSize: CGFloat!
-
-    /// Notifies the PillButton to refresh its UI to reflect its design token values
-    var themeDidUpdate: (() -> Void)?
-
-    var style: PillButtonStyle
-
-    init(style: PillButtonStyle) {
-        self.style = style
+/// Represents the set of `DynamicColor` values for the various states of a `PillButton`
+public struct PillButtonDynamicColors {
+    public init (rest: DynamicColor,
+                 hover: DynamicColor? = nil,
```

Comment:
Why are some of these nilable and some are not?

Reply:
I think to have a functional customized pill button, you really only need to define the rest, selected, disabled, and selected disabled states. Hover, highlighted, and selectedHighlighted seem to just be copies of rest or selected, even in our definitions.

Reply:
I see. Do you think it would be reasonable to just remove those colors entirely, especially since I don't see them called out in any specs? Or would that get too invasive?

Reply:
I'm not opposed, but I think the fall out from that would more or less also remove the custom pointer interaction methods of the bar. Which I guess means they probably aren't doing too much at the moment. If it doesn't break pointer interaction, I can remove the extra colors.

Reply:
Pointer interaction is still fine.